apps/showcase/src/main/java/org/apache/struts2/showcase/tiles/TilesAnnotationsAction.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.struts2.showcase.tiles;
+
+import org.apache.struts2.convention.annotation.Namespace;
+import org.apache.struts2.convention.annotation.ParentPackage;
+import org.apache.struts2.convention.annotation.Result;
+import org.apache.struts2.tiles.annotation.TilesDefinition;
+import org.apache.struts2.tiles.annotation.TilesPutAttribute;
+
+import com.opensymphony.xwork2.ActionSupport;
+
+@Namespace("/tiles")
+@ParentPackage("tiles")
+@Result(name = "success", type="tiles")
+@TilesDefinition(extend = "showcase.annotations", putAttributes = {
+        @TilesPutAttribute(name = "header", value = "/WEB-INF/tiles/header.jsp"),
+        @TilesPutAttribute(name = "body", value = "/WEB-INF/tiles/body.ftl"), })

```

Comment:
I think the last comma isn't needed and also what do you think about formatting like this:
```java
@Namespace("/tiles")
@ParentPackage("tiles")
@Result(name = "success", type="tiles")
@TilesDefinition(extend = "showcase.annotations", putAttributes = {
    @TilesPutAttribute(name = "header", value = "/WEB-INF/tiles/header.jsp"),
    @TilesPutAttribute(name = "body", value = "/WEB-INF/tiles/body.ftl")
})
```
?